src/test/java/edu/cornell/kfs/sys/util/FavoriteAccountLineBuilderTest.java

```diff
@@ -120,16 +118,23 @@ public static void setUp() throws Exception {
         GlobalVariables.setMessageMap(new MessageMap());
     }
 
-    private static <T extends Document> T getMockedDocument(Class<T> documentClass) {
-    	Pattern nonMockedMethodsPattern = Pattern.compile("^(get|set).*$");
-        List<String> methodNames = new ArrayList<>();
-        for (Method method : documentClass.getMethods()) {
-            if (!Modifier.isFinal(method.getModifiers()) && !nonMockedMethodsPattern.matcher(method.getName()).matches()) {
-                methodNames.add(method.getName());
-            }
-        }
-        IMockBuilder<T> builder = EasyMock.createMockBuilder(documentClass).addMockedMethods(methodNames.toArray(new String[0]));
-        return builder.createNiceMock();
+    private static CuRequisitionDocument buildMockCuRequisitionDocument() {
+        PowerMockito.suppress(PowerMockito.constructor(LedgerPostingDocumentBase.class));

```

Comment:
This exact same suppression is being called by all three of the document-building methods in this class. Would it be possible to perform the suppression in a "Before" or "BeforeClass" annotated method? Or is it necessary to suppress this right before the constructor call?

Reply:
I was able to move the suppress constructor call into the setup method, good idea